src/main/java/net/aufdemrand/denizen/events/player/PlayerPlacesHangingScriptEvent.java

```diff
@@ -94,12 +94,17 @@ public ScriptEntryData getScriptEntryData() {
     }
 
     @Override
-    public HashMap<String, dObject> getContext() {
-        HashMap<String, dObject> context = super.getContext();
-        context.put("hanging", hanging);
-        context.put("cuboids", cuboids);
-        context.put("location", location);
-        return context;
+    public dObject getContext(String name) {
+        if (name.equals("hanging")) {
+            return hanging;
+        }
+        else if (name.equals("cuboids")) {

```

Comment:
add "// Deprecated in favor of context.location.cuboids" to the end

`else if (name.equals("cuboids")) { // Deprecated in favor of context.location.cuboids`